deb/axolotl.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Type=Application
+Name=Axolotl
+GenericName=Signal Chat Client
+Path=/usr/share/axolotl/
+Exec=axolotl
+#Exec=axolotl -e qt
```

Comment:
As I see it, this is commented out. For what purpose is it there?
I would say we either remove it straight away, or we add a comment explaining when and why it will be needed.

Reply:
It was an idea to have the qt-option prepared for users in the desktop file, so that they don't have to create a new one. But this kind of "service" may be ommited for a better understanding - and then add an explanation, how to use the qt version somewhere else.

Reply:
Of course! That makes sense. What about adding what you just said as a comment in the file?

Reply:
Will do!